samples/ftd_configuration/physical_interface_with_default_dhcp.yml

```diff
@@ -0,0 +1,52 @@
+- hosts: all
```

Comment:
If I'm not wrong, the main part of this playbook is to remove default DHCP server. Does it make sense to rename it to smth like `dhcp_container_with_no_default_dhcp_server.yml`?

Reply:
makes sense as well.